dwh/indicator_lib.py

```diff
@@ -83,54 +99,37 @@ def calc_TX_PVLS(patient_agg_obs: pandas.DataFrame, VL_code: str,
     end_date = datetime.strptime(end_date_str, '%Y-%m-%d')
   temp_df = patient_agg_obs[(patient_agg_obs['code'] == VL_code)].copy()
   # Note the above copy is used to avoid setting a new column on a slice next:
-  # separate value from date
-  temp_df[['vl_max_date', 'latest_vl_value']] = temp_df.max_date_value.str.split(query_lib.DATE_VALUE_SEPARATOR,
-                                                                                       expand=True)
-  temp_df['latest_vl_value'] = temp_df.loc[:, 'latest_vl_value'].astype(float)
+  temp_df[['vl_max_date', 'latest_vl_value']] = temp_df.max_date_value.str\
+    .split(query_lib.DATE_VALUE_SEPARATOR, expand=True)
+  temp_df['latest_vl_value'] = temp_df['latest_vl_value'].astype(float)
```

Comment:
nit: We should probably use a different column name instead of having the same column on both sides of the assignment.